### PR TITLE
Added resolution.scss

### PR DESCRIPTION
**Pull request by dam-orzel**

### CODE REVIEW

File: src/assets/style/variables/resolution.scss [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    min-width: 1200px
+  )
+);
+@mixin res($name) {
+  @if map-has-key($breakpoints, $name) {
+    @media #{inspect(map-get($breakpoints, $name))} {
+      @content;
+    }
+  } @else {
+    @warn "Unfortunately, this value does not exist in resolution.scss `#{$breakpoint}`. "
```
Comment (hayuna) [inline src/assets/style/variables/resolution.scss:18]: I have a question about that warning. What will be steps to reproduce it ? I tried resizing my window and it didn't appear

Comment (dam-orzel) [reply src/assets/style/variables/resolution.scss:18]: Use it in style and write a wrong $breakpoint name. Than an error appear

Comment (hayuna) [reply src/assets/style/variables/resolution.scss:18]: Could I ask you to provide an example of code?

Comment (dam-orzel) [reply src/assets/style/variables/resolution.scss:18]: image

**Review by hayuna — APPROVED** [review]

**Pull request closed without merging**